src/robot.js

```diff
@@ -49,7 +49,23 @@ class Robot {
       response: new Middleware(this),
       receive: new Middleware(this)
     }
-    this.logger = new Log(process.env.HUBOT_LOG_LEVEL || 'info')
+    process.env.LOG_LEVEL = process.env.LOG_LEVEL || process.env.HUBOT_LOG_LEVEL || 'notice'
+    this.logger = {
+      debug () {
+        return log.notice(...arguments)
+      },
+      info () {
+        return log.info(...arguments)
+      },
+      warning () {
+        return log.warn(...arguments)
+      },
+      error () {
+        return log.error(...arguments)
+      }
+    }
```

Comment:
I'd suggest reducing this to:

```js
this.logger = log
```

Aliasing `debug` to `notice` makes it visible by default, but removes the ability to use the granularity of logging at debug level; I'm not sure that's the right thing to do. Is that a change here or to preserve historic behaviour?

Aliasing `warning` to `warn` is [built into that log package](https://github.com/medikoo/log#available-log-levels). (In fact we're pointing `warning` to the [alias `warn`](https://github.com/medikoo/log/blob/1448c93463b8ed1dbc804dcf80dbed147a50472c/lib/private/logger-prototype/index.js#L73-L76) here.)

Reply:
I tried setting `this.logger = log` but the tests failed. Sinon complained it couldn't wrap on undefined properties. Which is why I used this technique. I'll try again to see if I missed anything.

Reply:
I've implemented your suggestion and changed the test assertion strategy to pass.

Reply:
We can merge this as soon as you confirm the changes.